assets/js/atomic/blocks/product-elements/rating-counter/index.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { isExperimentalBuild } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+import sharedConfig from '../shared/config';
+import { supports } from './support';
+import { BLOCK_ICON } from './constants';
+
+if ( isExperimentalBuild() ) {
+	registerBlockType( metadata, {
+		...sharedConfig,
+		ancestor: [
+			'woocommerce/all-products',
+			'woocommerce/single-product',
+			'core/post-template',
+			'woocommerce/product-template',
```

Comment:
Is it intended that it can be used in Products block? It does not render anything in Editor and frontend and seems to work only in Single Product block. Or the others will be covered later?

View from Editor in Products (Beta) block.
<img width="761" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/657c5ba0-d89b-406b-914f-f606b25d7d33">

Reply:
Fixed it to work on other product blocks, I'm excluding it from the All Products for now.